package.json

```diff
@@ -259,13 +259,17 @@
 		"whybundled": "1.4.2",
 		"yargs": "13.3.0"
 	},
+	"resolutions": {
```

Comment:
This feature is necessary because `@wordpress/components@9.0.0` pins the version of `@emotion/core` resulting in duplicate versions and causing problems with context (and therefore theming).
https://unpkg.com/@wordpress/components@9.0.0/package.json

It appears to have been fixed in the latest version of `@wordpress/components` but is yet-to-be released. https://github.com/WordPress/gutenberg/blob/master/packages/components/package.json#L28

Reply:
So then why does it work on master? Because of the local emotion?

Reply:
Because we had `@emotion/core` pinned everywhere to the same version that `@wordpress/components` was using.